appholder/src/main/java/com/android/identity/wallet/support/AndroidSecureAreaSupport.kt

```diff
@@ -0,0 +1,135 @@
+package com.android.identity.wallet.support
+
+import androidx.biometric.BiometricPrompt
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.android.identity.android.securearea.AndroidKeystoreSecureArea
+import com.android.identity.android.securearea.KeystoreUtil
+import com.android.identity.securearea.SecureArea
+import com.android.identity.wallet.composables.AndroidSetupContainer
+import com.android.identity.wallet.composables.AuthenticationKeyCurveAndroid
+import com.android.identity.wallet.composables.MdocAuthentication
+import com.android.identity.wallet.composables.state.AndroidAuthKeyCurveOption
+import com.android.identity.wallet.composables.state.AndroidAuthKeyCurveState
+import com.android.identity.wallet.composables.state.AuthTypeState
+import com.android.identity.wallet.composables.state.MdocAuthOption
+
+class AndroidSecureAreaSupport(
+    private val capabilities: KeystoreUtil.DeviceCapabilities
```

Comment:
I know it's a longer word but I think calling it `AndroidKeystoreSecureAreaSupport` is better and also more precise. Ditto for `AndroidKeystoreSecureAreaSupportState`.